servers/quarkus-tests/src/main/java/org/projectnessie/quarkus/tests/profiles/NessieDockerTestResourceLifecycleManager.java

```diff
@@ -66,17 +67,20 @@ public void setIntegrationTestContext(DevServicesContext context) {
 
   @Override
   public void init(Map<String, String> initArgs) {
-    NessieContainer.NessieConfig.Builder configBuilder =
-        NessieContainer.builder()
-            .fromProperties(initArgs)
-            .keycloakContainerSupplier(KeycloakTestResourceLifecycleManager::getKeycloak);
-    context.containerNetworkId().ifPresent(configBuilder::dockerNetworkId);
-    containerConfig = configBuilder.build();
+    containerConfig = NessieContainer.builder().fromProperties(initArgs);
+    context.containerNetworkId().ifPresent(containerConfig::dockerNetworkId);
   }
 
   @Override
   public Map<String, String> start() {
-    nessie = containerConfig.createContainer();
+    CustomKeycloakContainer keycloak = KeycloakTestResourceLifecycleManager.getKeycloak();
```

Comment:
If wonder if we could break the dependency here too, maybe have the 2 resources communicate through system properties... we can look into that later though.